### ISSUE TITLE
Order comparison results by baseline similar to benchmark-ips

### ISSUE THREAD

**Issue by technicalpickles** [michaelherold/benchmark-memory#26]
When I'm writing benchmarks, I almost always want to compare against an existing result. [benchmark-ips](https://github.com/evanphx/benchmark-ips) supports this with:

```
Benchmark.ips do |x|
  # ...
  x.compare! order: :baseline
end
```

I'd love to have something similar for benchmark-memory, so that the the comparison values (ie 1.54x)  is relative to the first defined report.

**Comment by michaelherold** [comment]
Yes, this is a great idea. I'd be happy to review a PR if you want to put one together. Otherwise, I will do this the next time I have some free programming time.

It should have the same API as benchmark-ips, as you noted.

**Comment by michaelherold** [comment]
@technicalpickles, would you give the branch a try before I merge and release?

**Comment by wojtekLs** [comment]
GO dicks

SSSH fuckers
